packages/core/src/components/ozon-content/nodes/figuur.node.tsx

```diff
@@ -71,13 +81,13 @@ export class OzonContentFiguurNode implements OzonContentNode {
             }
           : undefined;
 
-      return (
+      const ozonFiguur = (
```

Comment:
fyi: https://stenciljs.com/docs/templating-jsx#avoid-shared-jsx-nodes

Ik kan niet zo snel inschatten of `ozonFiguur` nu vaker wordt gebruikt (ik denk het niet), maar dit ter info